plugins/CoreHome/templates/_topBar.twig

```diff
@@ -13,12 +13,17 @@
             {{ menu._html|raw }}
         {% else %}
             <a {% if menu._tooltip is defined %}title="{{ menu._tooltip }}"{% endif %}
-               id="topmenu-{{ menu._url.module|lower }}" target="_self"
-               href="index.php{{ menu._url|urlRewriteWithParameters }}" tabindex="3">{{ _self.menuItemLabel(label, icon) }}</a>
+               {% if menu._url.module is defined %}id="topmenu-{{ menu._url.module|lower }}"{% endif %}
+               {% if menu._url.module is defined %}
```

Comment:
not sure if this change is really needed?

Reply:
This change is required for the external link to https://matomo.org/mobile/ [plugins/API/Menu.php](https://github.com/matomo-org/matomo/pull/12780/files#diff-814cad470a873bc49eb27ba5c081f325).

Reply:
However, there is no need to duplicate the same if condition, so I have combined the two.